db/seeds.rb

```diff
@@ -96,4 +124,20 @@
   end
   puts "\n"
 end
-puts 'Seeding user items complete!'
+puts "Seeding user items complete!\n\n"
+
+# Seed 'StoreItems' table
+puts 'Creating store items with prices...'
+Item.all.each do |item|
+  Store.all.each do |store|
+    dollars = (0.0..9.0).step(1).to_a.sample
+    cents = ((10.0..90.0).step(10).to_a.sample + 9.0)/100
+    StoreItem.create!(
+      store_id: store.id,
+      item_id: item.id,
+      price: dollars + cents
+    )
+    puts "  Created #{StoreItem.count} store items" if StoreItem.count % 40 == 0
```

Comment:
whats the purpose of the `count % 40 == 0` condition ?

Reply:
The purpose is to display progress.
When the seed file is run, it prints out 40, 80, 120, etc. instead of 1, 2, 3, 4, 5, etc.